cmd/kubeadm/app/phases/addons/dns/manifests.go

```diff
@@ -26,6 +26,7 @@ metadata:
     k8s-app: kube-dns
     kubernetes.io/cluster-service: "true"
     kubernetes.io/name: "CoreDNS"
+    app.kubernetes.io/name: "coredns"
```

Comment:
We could register a brand new label to use instead. It depends what we want to declare, of course.

Reply:
I would personally stick with the recommended label and not create brand-new ones. At least for our use case, it is important that all different workloads possess one standard label